### ISSUE TITLE
Audio playlists are empty on DLNA devices

### ISSUE THREAD

**Issue by egg02** [jellyfin/jellyfin-ios#432]
Using audio playlists (m3u) works fine when using the jellyfin app (iOS, Web) for playback. But they do not work via DLNA. It does not matter if the DLNA client is an app (like Evermusic Pro on iOS) or a hardware device. The playlists are shown, but they are empty.

Note: albums and other collections are played correctly via DLNA

 - iOS Version: 16.4
 - Jellyfin Expo version: 34.0.0
 - Jellyfin Server version: 10.8.10

**Comment by thornbill** [comment]
I'm not sure exactly what you are referencing. Could you provide a screenshot? Thanks!

**Comment by egg02** [comment]
A screenshot won‘t help here.

I am using some audio players (hardware devices; two internet radios and one AVR) for DLNA audio playback.

This works fine with albums, but playlists are empty and cannot be played. There is no error message.

However, the web audio player and the Jellyfin app have no problem with playlists.

If you don‘t have an audio player, you can also test it with any software DLNA player.

**Comment by egg02** [comment]
Maybe this is related:

A playlist created within Jellyfin (by adding songs manually) is not even displayed on DLNA clients.

**Comment by anthonylavado** [comment]
It sounds like this belongs in the server repository? This one (expo) would be if you had a specific issue with the app on the iPhone - like if it didn't play some media or crashed or whatever. The _server_ side is what handles sending content to DLNA clients.

**Comment by egg02** [comment]
Yes, it is mainly a server issue. I am unable to play playlists via DLNA on, for instance, internet radio devices or other network receivers. This has nothing to do with the app.

On the other hand playback of playlists stored on the server (m3u) via the app is also not very stable. But this may also caused by the server.

Interestingly, playback of albums works perfectly on any device.

**Comment by thornbill** [comment]
Closing since this is unrelated to the iOS app.
